java/src/by/gsu/epamlab/task3/Runner.java

```diff
@@ -0,0 +1,69 @@
+package by.gsu.epamlab.task3;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Runner {
+
+    public static void main(String[] args) throws IOException {
+
+        Path path = Path.of("java/src/in.txt");
```

Comment:
You can create the file in the same directory as Runner and use FileReader

Reply:
yes, it's just another way to perform